### PR TITLE
[12.0][IMP] connector_jira: restrict linked AAL modification

### PR DESCRIPTION
**Pull request by alexey-pelykh**

### CODE REVIEW

**Comment by alexey-pelykh** [comment]
@guewen @simahawk please review

**Review by guewen — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
@simahawk bump 💯

File: connector_jira/models/account_analytic_line/common.py [hunk 1]
```diff
@@ -171,6 +180,68 @@ def _compute_jira_references(self):
...
+                    project_id.mapped('jira_bind_ids')._is_linked():
+                raise exceptions.UserError(_(
+                    'Timesheet can not be created in project linked to JIRA!'
+                ))
+
+    @api.multi
+    def _connector_jira_write_validate(self, vals):
+        if not self.env.context.get('connector_jira') and \
+                any(f in vals
+                    and normalize_field_value(self._fields[f], vals[f]) != x[f]
```
Comment (simahawk) [inline connector_jira/models/account_analytic_line/common.py]: not easy to read IMO

Comment (alexey-pelykh) [reply connector_jira/models/account_analytic_line/common.py]: Do you think it would worth extracting that?

Comment (simahawk) [reply connector_jira/models/account_analytic_line/common.py]: just split it in blocks (eg: if + nested for loops)

Comment (alexey-pelykh) [reply connector_jira/models/account_analytic_line/common.py]: Done

**Review by simahawk — APPROVED** [review]

**Review by artemsenko — APPROVED** [review]

**Comment by alexey-pelykh** [comment]
@guewen I guess good-to-merge?

**Comment by artemsenko** [comment]
Functional test LGTM

**Pull request merged**